roundabout-core/src/main/java/at/fhv/itm3/s2/roundabout/entity/RoundaboutCar.java

```diff
@@ -98,8 +100,19 @@ public double getTimeToTraverseSection(Street section) {
 
     @Override
     public double getTransitionTime() {
-        //TODO getTransitionTime()
-        return 3;
+        final double currentTime = getRoundaboutModel().getCurrentTime();
+        final double minPossibleTransitionTime = getTimeToTraverseCurrentSection();
+        final double carLastUpdateTime = getLastUpdateTime();
+
+        if ((currentTime - carLastUpdateTime) > minPossibleTransitionTime) {
+            final double standardCarAccelerationTime = getRoundaboutModel().getStandardCarAccelerationTime();
+            final double carAccelerationFactor = getDriverBehaviour().getAccelerationFactor();
+            final double carAccelerationTime = standardCarAccelerationTime * carAccelerationFactor;
+
```

Comment:
you could extract the three lines into a private? method which return the calculated car acceleration time